pkg/netdevice/netResourcePool.go

```diff
@@ -72,17 +72,36 @@ func (rp *netResourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.Devic
 // StoreDeviceInfoFile stores the Device Info files according to the
 //  k8snetworkplumbingwg/device-info-spec
 func (rp *netResourcePool) StoreDeviceInfoFile(resourceNamePrefix string) error {
+	var devInfo nettypes.DeviceInfo
 	for id, dev := range rp.GetDevicePool() {
 		netDev, ok := dev.(types.PciNetDevice)
 		if !ok {
 			return fmt.Errorf("StoreDeviceInfoFile: Only pciNetDevices are supported")
 		}
-		devInfo := nettypes.DeviceInfo{
-			Type:    nettypes.DeviceInfoTypePCI,
-			Version: nettypes.DeviceInfoVersion,
-			Pci: &nettypes.PciDevice{
-				PciAddress: netDev.GetPciAddr(),
-			},
+
+		vdpaDev := netDev.GetVdpaDevice()
+		fmt.Printf("VDPA DEVICE %+v \n", vdpaDev)
```

Comment:
I know its still WIP, just reminder to remove there `Printf`

Reply:
Thanks. Done